client/app/telecom/telephony/billingAccount/administration/optionsGroup/translations/Messages_fr_FR.xml

```diff
@@ -2,13 +2,13 @@
 <translations>
    <translation id="telephony_options_group_title" qtlid="334400">Options du groupe</translation>
    <translation id="telephony_billing_account_administration_options_group_hiddenExternalNumber" qtlid="456804">Relevé de consommations</translation>
-   <translation id="telephony_billing_account_administration_options_group_hiddenExternalNumber_label" qtlid="456817">Masquer les appels entrants / sortants&#160;:</translation>
+   <translation id="telephony_billing_account_administration_options_group_hiddenExternalNumber_label">Masquer les appels entrants / sortants</translation>
    <translation id="telephony_billing_account_administration_options_group_overrideDisplayedNumber" qtlid="456830">Présentation numéro interne/externe</translation>
-   <translation id="telephony_billing_account_administration_options_group_overrideDisplayedNumber_info" qtlid="456843">Trés utile dans une entreprise. Présentez le numéro du standard pour vos appels extérieurs et présentez votre propre numéro pour les appels internes à l’entreprise.</translation>
-   <translation id="telephony_billing_account_administration_options_group_overrideDisplayedNumber_label" qtlid="456856">Utiliser un affichage distinct&#160;:</translation>
+   <translation id="telephony_billing_account_administration_options_group_overrideDisplayedNumber_info">Présenter le numéro du standard pour les appels vers l'extérieur et votre propre numéro pour les appels internes.</translation>
+   <translation id="telephony_billing_account_administration_options_group_overrideDisplayedNumber_label" qtlid="456856">Utiliser un affichage distinct</translation>
```

Comment:
You should remove the qtlid else the "&#160;" will not be removed for english and other languages